docs/core/tutorials/using-on-mac-vs-full-solution.md

```diff
@@ -1,13 +1,13 @@
 ---
 title: Building a complete .NET Core solution on macOS using Visual Studio for Mac
-description: This topic walks you through building a .NET Core solution that includes a reusable library and unit testing.
+description: This article walks you through building a .NET Core solution that includes a reusable library and unit testing.
 author: mairaw
-ms.date: 06/12/2017
+ms.date: 12/19/2019
 ms.custom: seodec18
 ---
 # Building a complete .NET Core solution on macOS using Visual Studio for Mac
```

Comment:
```suggestion
# Build a complete .NET Core solution on macOS using Visual Studio for Mac
```

Reply:
Actually "Create" might be a better verb, so as not to confuse with "compile".